src/core/crypto/poseidon.ts

```diff
@@ -98,10 +98,13 @@ export function bytesToBigIntLE(bytes: Uint8Array): bigint {
   return result;
 }
 
-export function bigIntToBytesLE(value: bigint, length: number): Uint8Array {
+export function bigIntToBytesLE(value: bigint | number, length: number): Uint8Array {
```

Comment:
Do we have correctness tests to ensure that a number is encoded correctly as LE?

I see there's a test that converts using these two functions nested in each other, but that just proves that they're inverses of each other, but not necessarily that the middle format is correct.

Reply:
done